### PR TITLE
Fix crossOriginIsolated status for Chrome

### PR DESCRIPTION
**Pull request by Niek**
See: https://www.chromestatus.com/feature/5690888397258752

### CODE REVIEW

File: api/WindowOrWorkerGlobalScope.json [hunk 1]
```diff
@@ -570,10 +570,10 @@
...
             "chrome": {
-              "version_added": false
+              "version_added": "87"
             },
             "chrome_android": {
-              "version_added": false
+              "version_added": "87"
             },
             "edge": {
               "version_added": false
```
Comment (ddbeck) [inline api/WindowOrWorkerGlobalScope.json:579]: suggestion: mirror to other Chromium-derived browsers

According to the [intent to ship](https://groups.google.com/a/chromium.org/g/blink-dev/c/cJ5dXIcQCsc) and some direct testing of my own, this shipped on all of these browsers except WebView. Can we set Edge to 87 and Opera (desktop) to 73?

```suggestion
              "version_added": "87"
```

**Review by ddbeck — CHANGES_REQUESTED** [review]
Hi @Niek and thank you for this PR. I see that it's your first on the project—welcome!

This looks good, but I have one suggestion (see line comment) to fill out the data for the other Chromium-derived browsers. Thank you again!

File: api/WindowOrWorkerGlobalScope.json (hunk 1 above)
Comment (Niek) [reply api/WindowOrWorkerGlobalScope.json]: Thanks for the suggestion @ddbeck - I have updated the PR now.

**Review by ddbeck — APPROVED** [review]
Looks great. Thank you! 🎉

**Pull request merged**